### PR TITLE
Fix dangling references to React Router + gatsbyjs.org fixes for @reach/router

### PR DESCRIPTION
**Pull request by KyleAMathews**

### CODE REVIEW

**Comment by KyleAMathews** [comment]
Deploy preview for *using-postcss-sass* failed.

Built with commit fe31c7006056e13181ed68188aa6cba589573ae5

https://app.netlify.com/sites/using-postcss-sass/deploys/5b697b8b3813f010e521bb11

File: www/src/utils/react-router-state-manager.js [hunk 1]
```diff
@@ -1,26 +1,27 @@
 import React from "react"
```
Comment (m-allanson) [inline www/src/utils/react-router-state-manager.js]: change this filename?

**Comment by KyleAMathews** [comment]
Deploy preview for *gatsbyjs* failed.

Built with commit fe31c7006056e13181ed68188aa6cba589573ae5

https://app.netlify.com/sites/gatsbyjs/deploys/5b697b893813f010e521bb05

File: packages/gatsby/README.md [hunk 2]
```diff
@@ -1,107 +1,163 @@
-<img alt="Gatsby" src="https://www.gatsbyjs.org/monogram.svg" width="100">
+<p align="center">
```
[the remark this answers is not in the document]
Comment (KyleAMathews) [reply packages/gatsby/README.md]: We could perhaps symlink it — I vaguely remember weird things happening though with them so copying is probably easier.

File: www/src/utils/react-router-state-manager.js (hunk 1 above)
Comment (KyleAMathews) [reply www/src/utils/react-router-state-manager.js]: 👍

**Comment by KyleAMathews** [comment]
Deploy preview for *using-remark* failed.

Built with commit a1ace9278d92d0ef106f309d669614332c464cf4

https://app.netlify.com/sites/using-remark/deploys/5b698bb902ed8323b3bbd727

**Pull request merged**